pkg/testers/clusterloader2/cl2.go

```diff
@@ -94,7 +96,7 @@ func (t *Tester) Test() error {
 	// TODO(amwat): get prebuilt binaries
 	cmd := exec.Command("go", append(cmdArgs, args...)...)
 	exec.InheritOutput(cmd)
-	cmd.SetDir(filepath.Join(t.RepoRoot, "clusterloader2"))
```

Comment:
we still need this. This is the source directory https://github.com/kubernetes/perf-tests/tree/master/clusterloader2 not the report directory.

Reply:
Changed this back to the original value. Tx for that @amwat .